packages/babel-plugin/src/utils/label.js

```diff
@@ -53,6 +53,21 @@ export function getLabelFromPath(path: *, state: *, t: *) {
   )
 }
 
+const getObjPropertyLikeName = (path, t) => {
+  if (
+    (!t.isObjectProperty(path) && !t.isObjectMethod(path)) ||
+    path.node.computed
+  ) {
+    return null
+  }
+  if (t.isIdentifier(path.node.key)) {
+    return path.node.key.name
+  } else {
+    // literals
+    return String(path.node.key.value).replace(/\s+/g, '-')
```

Comment:
This handles numeric and bigint literals but I think that I should revert this and only handle string literals here. Thoughts?

Reply:
Yeah, I'd just do string literals

Reply:
done